src/Compilers/VisualBasic/Test/CommandLine/ErrorLoggerTests.vb

```diff
@@ -0,0 +1,202 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Globalization
+Imports System.IO
+Imports Microsoft.CodeAnalysis.Test.Utilities
+Imports Microsoft.CodeAnalysis.Test.Utilities.SharedResourceHelpers
+Imports Microsoft.CodeAnalysis.VisualBasic.UnitTests
+Imports Xunit
+
+Imports Microsoft.CodeAnalysis.CommonDiagnosticAnalyzers
+Imports Microsoft.CodeAnalysis.DiagnosticExtensions
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.CommandLine.UnitTests
+
+    Public Class ErrorLoggerTests
+        Inherits BasicTestBase
+
+        Private Shared ReadOnly libDirectory As String = Environment.GetEnvironmentVariable("LIB")
```

Comment:
I just shamelessly cloned this from http://source.roslyn.io/#Roslyn.Compilers.VisualBasic.CommandLine.UnitTests/TouchedFileLoggingTests.vb,3f3d801a4ec8543e 
I'll clean up both the test files.